package.json

```diff
@@ -22,6 +22,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "node ./build-non-split.js",
+    "prepare": "cp build/static/js/main.js dist/ && cp build/static/css/main.*.css dist/",
```

Comment:
Should we create the dist directory if it doesn't exist/remove & replace it if it does?

Reply:
I'm not too worried because the `/dist/` directory can be in the github repo. I can certainly iterate on this, though, but I'd say we don't need it yet.